src/spawn_stubs.c

```diff
@@ -21,6 +22,22 @@
 #include <pthread.h>
 #include <signal.h>
 
+#if defined(__APPLE__)
```

Comment:
This whole block is under `#if !defined(_WIN32)` (on line 12), so we don't end up defining `spawn_is_osx` on win32, as CI helpfully points out. I guess we should simply move this block up (or down, after the big conditional block).

Reply:
Ooops, fixing that